feedparser/datetimes/rfc822.py

```diff
@@ -91,6 +91,10 @@ def _parse_date_rfc822(date):
         parts.extend(("00:00:00", "0000"))
     # Remove the day name
     if parts[0][:3] in day_names:
+        # Comma without spaces:
+        # 'Fri,24 Nov 2023 18:28:36 -0000'
+        if "," in parts[0] and parts[0][-1] != ",":
+            parts.insert(1, parts[0].split(",")[1])
```

Comment:
Considering the case of double commas, the behavior of `if "," in parts[0] and parts[0][-1] != ",":` maybe undesired when the date is `Fri,, 24 Nov 2023 18:28:36 -0000` (there is a space after the double commas).

So it may be better to do `rpartition()` first:
```python
after_comma = parts[0].rpartition(",")[2]
if after_comma:
    parts.insert(1, after_comma)
```